promcw/promcw.go

```diff
@@ -390,7 +390,7 @@ func (b *Bridge) fetchMetricFamilies(ch chan<- *dto.MetricFamily) {
 		client := &http.Client{Transport: transport}
 		resp, err = client.Do(req)
 		if err != nil {
-			logger.Errorf("url=%q, err=[%v]", url, err.Error())
+			logger.KV(xlog.ERROR, "url", "err", url, err.Error())
```

Comment:
fix for all in format:
`"key1", value1, "key2", val2`, etc

`logger.KV(xlog.ERROR, "url", url, "err", err.Error())`